### PR TITLE
Merge '19-develop play screen UI' branch to development

### LINKED ISSUE #19: develop play screen UI
### develop the following UI widgets:
- [x] back button
- [x] pause button
- [x] visible "Gamemode" and "subtitle" labels
- [x] visible "score" and "time" labels
- [x] rectangular space for built words
- [x] text bar for words being built
- [x] keyboard (rectangular) space for each letter (button) to be scooped (pressed)
- [x] individual buttons for each letter to be scooped
- [x] backspace button

### PR DESCRIPTION
**Pull request by jiroblea**
### developed the following UI widgets:
- back button
- pause button
- visible "Gamemode" and "subtitle" labels
- visible "score" and "time" labels
- rectangular space for built words
- text bar for words being built
- keyboard (rectangular) space for each letter (button) to be scooped (pressed)
- individual buttons for each letter to be scooped
- backspace button

### CODE REVIEW

**Comment by jiroblea** [comment]
please see the attached image:

image

File: iskrambol/playscreen.kv [hunk 1]
```diff
@@ -9,26 +9,116 @@
...
+                    Button:
+                        text: 'R'
+
+                BoxLayout:
+                    orientation: 'horizontal'
+
+                    Button:
+                        text: 'E'
+                    Button:
+                        text: 'M'
```
Comment (Serv08) [inline iskrambol/playscreen.kv:124]: Can we use a function to make a dynamic box layout?

**Comment by Serv08** [comment]
Additional comment for the previous commit:
For the line 35 in `main.py`, why is `PlayScreen` class called from an import instead of creating a class directly in the `main.py`?

File: iskrambol/playscreen.kv (hunk 1 above)
Comment (jiroblea) [reply iskrambol/playscreen.kv:124]: Yes, we can use a function to make a dynamic layout. For the purpose of developing the UI fast, I used those BoxLayouts.

**Comment by jiroblea** [comment]
> For the line 35 in `main.py`, why is `PlayScreen` class called from an import instead of creating a class directly in the `main.py`?

Because `PlayScreen` class will contain a lot of code for the game. I think that the separated file for the class is needed to maintain organized code this early in the development.

**Comment by Serv08** [comment]
LGTM

**Review by Serv08 — APPROVED** [review]
LGTM

**Pull request merged**